src/main/scala/cs332/distributedsorting/slave/Slave.scala

```diff
@@ -15,10 +16,12 @@ object Slave {
   }
 
   def main(args: Array[String]): Unit = {
-    val client = Slave("localhost", 50051)
+    val client = Slave("localhost", 50061)
     try {
       val user = args.headOption.getOrElse("world")
-      client.greet(user)
+      val localhost: InetAddress = InetAddress.getLocalHost
+      val localIpAddress: String = localhost.getHostAddress
```

Comment:
What is the value of `loclaIpAddress` if you run the application at local machine?
Is it something like loopback ip address(`127.0.0.1`) or private ip address(`192.168.0.12`)? or public ip address(`141.223.12.34`)?

Reply:
I emailed to the professor and he said that it doesn't matter that ip address is private or public if it can be connected with each other.
So I think it is okay with any ip address of the available network interfaces which is not the loopback ip address(127.0.0.1).